targets/targets.json

```
@@ -7503,7 +7503,6 @@
         "core": "Cortex-M7",
         "supported_toolchains": [
             "GCC_ARM",
-            "IAR",
             "ARMC6"
```

```suggestion
            "ARM"
```
Since you touched IAR here.